src/js/actions/index.js

```diff
@@ -105,11 +105,14 @@ export function fetchUsers() {
 
 export function fetchSessionInfo() {
   const url = '/openboxes/api/getAppContext';
-  const request = apiClient.get(url);
-
-  return {
-    type: FETCH_SESSION_INFO,
-    payload: request,
+  return (dispatch) => {
```

Comment:
Why you refactored this action?

Reply:
because it is not how redux-thunk is supposed to work - I don't understand why we were passing request in dispatch instead of payload from this request - in some cases in turned out to be understood as "fetched" when I console.logged sessionReducer, but in reality the payload didn't come yet.

Reply:
@kchelstowski makes sense, but then we have to refactor all remaining actions too. So I think there should be at least a TODO combined with a tech debt ticket to not forget about it.